src/services/tech-record-document-conversion.ts

```diff
@@ -69,6 +70,7 @@ const upsertTechRecords = async (
   const techRecords = techRecordDocument.techRecord;
 
   if (!techRecords) {
+    debugLog(`there are no tech records - returning ...`);
```

Comment:
I think that this will be useful to have at the info level

Reply:
It looks like the repo leans more towards only console logging errors, all console.info logs are wrapped in the debugLog, probably to avoid spamming CW. So I'm not sure there is a need for us to deviate from that. 

If we want the logging we can quite easily turn it on.